### ISSUE TITLE
asking for gpu to finetune large model

### ISSUE THREAD

**Issue by kunci115** [huggingface/parler-tts#111]
can we do finetune large model with 1 gpu from 4090?

**Comment by kunci115** [comment]
also if i finetuned with singaporean english accent dataset, and not differentiate name of the speaker(put all voice training data in one folder and train it) will it still understand previous label name for example "jenny" ?
